src/topology/continuous_function/algebra.lean

```diff
@@ -422,6 +422,16 @@ by { rw algebra.algebra_map_eq_smul_one, refl, }
 
 variables {𝕜 : Type*} [field 𝕜] [topological_space 𝕜] [topological_ring 𝕜]
 
+/--
+A set of functions "separates points strongly"
+if for each pair of distinct points there is a function with specified values on them.
+
+We give a slightly unusual formulation, where the specified values are given by some
+function `v`, and we ask `f x = v x ∧ f y = v y`. This avoids needing a hypothesis `x ≠ y`.
+-/
+def set.separates_points_strongly (s : set C(α, 𝕜)) : Prop :=
+∀ (v : α → 𝕜) (x y : α), ∃ f : s, (f x : 𝕜) = v x ∧ f y = v y
```

Comment:
It seems a bit odd to put something in the set namespace that depends on `C(α, 𝕜)` without any mention of continuity in the docstring or name.

Reply:
I've add a paragraph to the doc-string explaining my reasoning for the unreasonable name. Let me know what you think.